data/ui/connect_panel.glade

```diff
@@ -580,6 +621,7 @@
       <widget name="login_password_entry"/>
       <widget name="remember_login_password_check_button"/>
       <widget name="login_cert_file_chooser_button"/>
+      <widget name="client_cert_file_chooser_button"/>
```

Comment:
The  client_cert name is coufusing, how about rename the login_cert to ecdsa_cert and this to another better name? such as sasl_external_cert?

Reply:
Yes, we can add a new confval like `login.ecdsa_certificate`, but keep it compatible with the old `login.certificate` confval.